.github/workflows/auto-updater.yml

```diff
@@ -0,0 +1,47 @@
+name: Automatic Data Update
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 * * *'
+
+jobs:
+  DataUpdater:
+    name: Automatic Data Update
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'sapphiredev'
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v4
+      - name: Use Node.js v20
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: yarn
+          registry-url: https://registry.yarnpkg.com/
+      - name: Install Dependencies
+        run: yarn --immutable
+      - name: Run plugin-api MIME type sync
+        working-directory: ./packages/api
+        run: yarn sync-mime-types
+      - name: Run prettier on the code
+        run: yarn format
+      - name: Commit any changes and create a pull request
+        env:
+          GITHUB_USER: github-actions[bot]
+          GITHUB_EMAIL: 41898282+github-actions[bot]@users.noreply.github.com
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git add .;
+          if git diff-index --quiet HEAD --; then
+            echo "No changes to commit, exiting with code 0"
+            exit 0;
+          else
+            git remote set-url origin "https://${GITHUB_TOKEN}:x-oauth-basic@github.com/${GITHUB_REPOSITORY}.git";
+            git config --local user.email "${GITHUB_EMAIL}";
+            git config --local user.name "${GITHUB_USER}";
+            git checkout -b update-iana-mime-type/$(date +%F-%H-%M);
+            git commit -sam "feat: update mime types";
+            git push --set-upstream origin $(git rev-parse --abbrev-ref HEAD)
+            gh pr create -t "feat: update mime types" -b "*bleep bloop* I updated the IANA mime type list" -B main;
```

Comment:
Yknow we could also make the CI enable automerge on the PR, as well as request reviews

Also is there _any_ reason we dont wanna just auto release these changes @favna?

Reply:
This is based on /utilities's [`auto-updater.yml`](https://github.com/sapphiredev/utilities/blob/main/.github/workflows/auto-updater.yml), which makes a PR.